useragent.gemspec

```
@@ -1,17 +1,18 @@
 Gem::Specification.new do |s|
-  s.name    = "useragent"
-  s.version = "0.16.11.pre.3b5e242"
+  s.name    = 'useragent'
+  s.version = '0.16.11.pre.3b5e242'
```

```suggestion
  s.version = '0.16.11'
```

We should not commit our `.pre.` suffixes